content/en/tracing/trace_collection/compatibility/java.md

```diff
@@ -98,7 +98,7 @@ Integrations in Preview are disabled by default but can be enabled individually:
 | Java Servlet Compatible | 2.3+, 3.0+ | Fully Supported                                        | `servlet`, `servlet-2`, `servlet-3`                      |
 | Jax-RS Annotations      | JSR311-API | Fully Supported                                        | `jax-rs`, `jaxrs`, `jax-rs-annotations`, `jax-rs-filter` |
 | Jetty                   | 7.0-12.x   | Fully Supported                                        | `jetty`                                                  |
-| Micronaut HTTP Server   | 2.x        | Fully Supported                                        | `micronaut`                                              |
+| Micronaut HTTP Server   | 2.x+        | Fully Supported                                        | `micronaut`                                              |
```

Comment:
This deletes extra spaces to keep the pipe characters aligned when viewed in Markdown. GitHub's diff below doesn't quite portray this but it should be clear in the overall diff once you commit it:
```suggestion
| Micronaut HTTP Server   | 2.x+       | Fully Supported                                       | `micronaut`                                             |
```